addons/edi/models/edi_record.py

```diff
@@ -122,6 +132,8 @@ class EdiRecord(models.AbstractModel):
         ondelete="cascade",
     )
 
+    error = fields.Char(string='Error', help='Records errors in execution')
```

Comment:
Evil single quotes.

Reply:
Reformatted entirely both edi and edi_sale now.